ecosystem/sep-0040.md

```diff
@@ -0,0 +1,160 @@
+## Preamble
+
+```
+SEP: To Be Assigned
+Title: Non-Fungible Token Standard
+Author: George Kudrayvtsev <george@stellar.org>
+Track: Standard
+Status: Draft
+Created: 2021-01-31
+Updated: 2021-01-31
+Version: 0.1.0
+Discussion: TBD
+```
+
+## Simple Summary
+This SEP defines a standard interface to describe non-fungible tokens (NFTs) on the Stellar network, including a distribution mechanism and metadata fields. It draws inspiration from the [EIP-721 standard](https://eips.ethereum.org/EIPS/eip-721).
+
+
+## Motivation
+A standard interface for NFTs allows ecosystem applications (such as wallets or marketplaces) to create, view, or otherwise work with any NFT on Stellar.
+
+
+## Abstract
+Non-fungible tokens (NFTs) can describe many types of assets, from digital collectibles to physical items. We should also define a standard issuance model to explain how NFTs can and should be issued on the network. In order to maintain compatibility across the Stellar ecosystem and beyond, we need a standard way to describe and define them that is agnostic of what the NFT actually describes.
+
+
+## Specification
+We begin with the [issuance model](#issuance-model) and then cover the NFT [metadata specification](#nft-metadata-specification).
+
+### Issuance Model
+Given an arbitrary digital asset, here's how issuance typically works:
+
+  1. An account is created specifically to represent the NFT in the Stellar ledger.
+  2. The account configures itself with metadata describing the NFT.
+  3. The account issues a new Stellar asset representing ownership over said NFT.
+  4. Any account holding that asset lays claim to the NFT.
```

Comment:
These steps feel too high-level for the specification. These steps could go into the abstract possibly, or you could keep it here, but if the protocol is specifying how to issue the asset, I think it should do so explicitly, by stating what transactions/operations need submitting, and what values to use, etc.

Reply:
Or what final state of expected if it's unimportant which operations are used when.

Reply:
Good idea! I've expanded the issuance model (incl. requiring a distribution account) in 37df0a5; wdyt?